gapis/server/server.go

```diff
@@ -125,9 +126,24 @@ func (s *server) LoadCapture(ctx context.Context, path string) (*path.Capture, e
 	return capture.Import(ctx, name, in)
 }
 
+// Returns all devices, sorted by Android first, and then Host
+func getSortedDevices(ctx context.Context) []bind.Device {
+	all := bind.GetRegistry(ctx).Devices()
+	filtered := make([]bind.Device, 0, len(all))
+	for _, dev := range all {
+		instance := dev.Instance()
+		if instance.GetConfiguration().GetOS().GetKind() == device.Android {
+			filtered = append([]bind.Device{dev}, filtered...)
```

Comment:
Uber nit: While the number of devices is small, this is acceptable, but this does not scale particularly well. Would be nicer to have two lists you append to, then join at the end.